### PR TITLE
MASSIVE CHANGES

### PR DESCRIPTION
**Pull request by PembleC**
This fixes almost all the problems.

### CODE REVIEW

**Comment by MannyB6** [comment]
This looks really nice. I like how the game looks when loaded up. Great job.

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 language: java
 
-jdk: oraclejdk10
+jdk: openjdk10
```
Comment (reinscha) [inline .travis.yml]: Good stuff here, now we can pass some test!

File: src/main/java/views/ApplicationController/index.ftl.html [hunk 2]
```diff
@@ -1,151 +1,137 @@
...
-        <p>8</p>
-        <p>9</p>
-        <p>10</p>
-        <!--<div class="titleAI">-->
-        <!--<h1>Opponent</h1>-->
-        <!--</div>-->
-    </div>
-    <div class="gridCol">
-        <pre>    A   B   C   D   E   F   G   H   I   J</pre>
+    <div id="userTitle" class="boardTitle">
```
Comment (reinscha) [inline src/main/java/views/ApplicationController/index.ftl.html]: Nice job! Look so much better now

File: .travis.yml (hunk 1 above)
Comment (PembleC) [reply .travis.yml]: You Know it!

**Pull request merged**